ci-operator/config/openshift/ocs-operator/openshift-ocs-operator-master__latest-upstream.yaml

```diff
@@ -0,0 +1,80 @@
+binary_build_commands: make build
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: rhel-8-release-golang-1.16-openshift-4.8
+images:
+- dockerfile_literal: FROM quay.io/ocs-dev/ocs-operator:latest
+  to: ocs-operator-master
+- dockerfile_literal: FROM ceph/daemon-base:latest-master
+  to: ceph-master
+- dockerfile_literal: FROM rook/ceph:master
+  to: rook-master
+- dockerfile_literal: FROM quay.io/ocs-dev/noobaa-core:master
+  to: noobaa-core-master
+- dockerfile_literal: FROM quay.io/ocs-dev/noobaa-operator:master
+  to: noobaa-operator-master
+operator:
+  bundles:
+  - as: dev-master-dependencies
+    base_index: noobaa-index
+    context_dir: deploy/
+    dockerfile_path: openshift-ci/Dockerfile.bundle.master
+  substitutions:
+  - pullspec: quay.io/ocs-dev/ocs-operator:.*
+    with: ocs-operator-master
+  - pullspec: ceph/daemon-base:.*
+    with: ceph-master
+  - pullspec: rook/ceph:.*
+    with: rook-master
+  - pullspec: noobaa/noobaa-core:.*
+    with: noobaa-core-master
+  - pullspec: noobaa/noobaa-operator:.*
+    with: noobaa-operator-master
```

Comment:
@jarrpa I'm curious what you think of this. Notably the bits below.

```yaml
- dockerfile_literal: FROM quay.io/ocs-dev/ocs-operator:latest
  to: ocs-operator-master
# ...
  - pullspec: quay.io/ocs-dev/ocs-operator:.*
    with: ocs-operator-master
```

Instead of building ocs-operator for the nightly builds, it just grabs the latest one that has already been built from quay. We could remove the FROM-only build and the substitution and just have the bundle use the existing `quay.io/ocs-dev/ocs-operator:latest`, but doing the "build" means that we can't have any case of the latest image changing during a test and effectively testing two different versions.

We could also go back to building the image from source, but that seems unnecessary to me.